terrawrap/utils/version.py

```diff
@@ -0,0 +1,104 @@
+"""Contains functions for checking the latest version of this package"""
+import sys
+import tempfile
+import os
+import json
+from typing import Dict, Any
+from datetime import datetime
+from time import sleep
+
+import requests
+from packaging import version
+
+
+def version_check(current_version: str):
+    """
+    Print a warning message if a stale version of Terrawrap is detected.
+    :param current_version: The currently installed version of Terrawrap.
+    :return: True if the version of Terrawrap is stale.
+    """
+    try:
+        latest_version = get_latest_version(current_version=current_version)
+        if version.parse(latest_version) <= version.parse(current_version):
+            return False
+
+        print(
+            "WARNING: Your version of Terrawrap is stale! You have version '%s' but the latest is '%s'" % (
+                current_version,
+                latest_version,
+            ),
+            "Please upgrade as soon as possible!\n",
+            sep="\n",
+            file=sys.stderr,
+        )
+        sleep(5)
```

Comment:
Is the sleep needed? Maybe make it shorter

Reply:
It's intentional to give people time to read the message and to annoy them into upgrading.